src/pages/ProcessBrowserAuth.vue

```diff
@@ -5,42 +5,21 @@
 <script>
 import store2 from 'store2';
 
-function checkHashSuccess(hash) {
-	if (hash.indexOf('error') > -1) {
-		return false;
-	}
-	if (hash.indexOf('access_token') === -1
-		&& hash.indexOf('id_token') === -1
-		&& hash.indexOf('refresh_token') === -1) {
-		return false;
-	}
-	return true;
-}
-
 export default {
 	name: 'ProcessBrowserAuth',
 	inject: ['kvAuth0'],
 	mounted() {
-		const hashKey = 'auth0.browser_hash';
+		const { state } = this.$route.query;
```

Comment:
Will this page just check called automatically with our current auth flow? I didn't see it being called explicitly in this change set.

Reply:
Yeah, this page is the current `redirectUri`